install/installer.class.php

```diff
@@ -230,7 +230,7 @@ protected function createFirstAccess() {
 
       $newRights = [
          PluginFlyvemdmProfile::$rightname         => PluginFlyvemdmProfile::RIGHT_FLYVEMDM_USE,
-         PluginFlyvemdmInvitation::$rightname      => CREATE | READ | DELETE | PURGE,
+         PluginFlyvemdmInvitation::$rightname      => CREATE | READ | UPDATE | DELETE | PURGE,
```

Comment:
use ALLSTANDARDRIGHT whics is a shortcut for CREATE | READ | UPDATE | DELETE | PURGE